### DISCUSSION TITLE
Is it possible to run onnx model on the camera?

### DISCUSSION THREAD

**Discussion by anigabriel** [AxisCommunications/acap-computer-vision-sdk-examples#41] [Q&A] [answered]
Is it possible to run .onnx model on AXIS camera P3265-LVE Dome?

**Comment by Corallo** [comment, marked as answer]
Hello anigabriel, thanks for your question.
At the moment no, not directly, you need to convert the model from onnx format to tflite.

**Comment by Corallo** [comment]
I am converting this issue into a Q&A discussion
